modules/cloud-function/main.tf

```diff
@@ -137,6 +137,8 @@ data "archive_file" "bundle" {
     ? "/tmp/bundle.zip"
     : var.bundle_config.output_path
   )
+  output_file_mode = var.bundle_config.output_file_mode
```

Comment:
Do we need a variable for `output_file_mode`. Would it make more sense to just use 0666?